syntax = "proto3";
package floorplan;

import "google/api/annotations.proto";


message ProcessRequest {
    bytes image_data = 1;
}


message ProcessResponse {
    bytes model = 1;
}


service Service {
    rpc Process (ProcessRequest) returns (ProcessResponse) {
        option (google.api.http) = {
            post: "/process"
        };
    }
}
